.rubocop.yml

```diff
@@ -41,14 +41,6 @@ Style/FormatStringToken:
     - 'lib/migrate_assets_to_asset_manager.rb'
     - 'lib/tasks/scheduled_publishing.rake'
 
-# Cops disabled, due to false positives and infinite loops
-
-Layout/AccessModifierIndentation:
-  Enabled: false
-
-Style/AccessModifierDeclarations:
-  Enabled: false
```

Comment:
I understand these changes based on the commit message, but there are a lot of other little changes in c3888444c022567ac6209ae3e503962b7d5965e4, could they be moved to a 'general tidy up' commit?

Reply:
Ah - all of those changes are because of those cops being re-enabled. I've amended the commit message to be clearer.

Reply:
Excellent 👍